src/main/java/com/example/java/test/junior/developer/configuration/SecurityConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.example.java.test.junior.developer.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+  private final RestTemplate restTemplate;
+  @Value("${external.api.url}")
+  private String externalApiUrl;
+
+  public SecurityConfig(RestTemplate restTemplate) {
+    this.restTemplate = restTemplate;
+  }
+
+  @Override
+  protected void configure(HttpSecurity http) throws Exception {
+    http.csrf().disable()
+        .authorizeRequests()
+        .antMatchers("/api/**").authenticated()
```

Comment:
Do you think this is correct? 
How about `/api/v1/login`, `/api/v1/logout`, and `POST /api/v1/users` (create user)? How can they be secured?